enlighten/Controller.py

```diff
@@ -1162,19 +1175,21 @@ def make_shortcut(kseq, callback):
             shortcut.activated.connect(callback)
             self.shortcuts[kseq] = shortcut
 
-        # techniques
-        make_shortcut("F1",     self.page_nav.set_technique_hardware)
-        make_shortcut("F2",     self.page_nav.set_technique_scope)
-        make_shortcut("F3",     self.page_nav.set_technique_raman)
-        make_shortcut("F4",     self.page_nav.set_technique_transmission)
-        make_shortcut("F5",     self.page_nav.set_technique_absorbance)
+        # views
+        make_shortcut("F1",     self.page_nav.set_view_hardware)
+        make_shortcut("F2",     self.page_nav.set_view_scope)
+        make_shortcut("F3",     self.page_nav.set_view_raman)
+        make_shortcut("F4",     self.page_nav.set_view_transmission)
+        make_shortcut("F5",     self.page_nav.set_view_absorbance)
 
         # operation modes
-        make_shortcut("F6",     self.page_nav.set_operation_mode_setup)
-        make_shortcut("Ctrl+1", self.page_nav.set_operation_mode_setup)
+        #make_shortcut("F6",     self.page_nav.set_operation_mode_setup)
+        make_shortcut("Ctrl+1", self.page_nav.set_operation_mode_raman)
+
+        #make_shortcut("F7",     self.page_nav.set_operation_mode_capture)
+        make_shortcut("Ctrl+2", self.page_nav.set_operation_mode_non_raman)
 
-        make_shortcut("F7",     self.page_nav.set_operation_mode_capture)
-        make_shortcut("Ctrl+2", self.page_nav.set_operation_mode_capture)
+        make_shortcut("Ctrl+3", self.page_nav.set_operation_mode_expert)
```

Comment:
Not sure about the shortcuts.  I kind of like having Ctrl+1 be Setup View, and Ctrl+2 be Scope View (or vice-versa).  (Or just Ctrl+1 be "Toggle View between Scope and Setup").  Soft clay either way.